helm/credentiald-chart/values.yaml

```diff
@@ -1,8 +1,30 @@
 deployment:
   replicas: 2
 
+image:
+  registry: "quay.io"
+  repository: "giantswarm/credentiald"
+  tag: "[[ .SHA ]]"
 pod:
   user:
     id: 1000
   group:
     id: 1000
+resource:
```

Comment:
How is this related to network policies?

Reply:
it's not exactly, but I was trying to standardise values.yml at the same time. I can do it in a separate PR if necessary. Which do you prefer?